src/components/panes/ConfirmImportPane.jsx

```diff
@@ -109,6 +109,10 @@ export default class ConfirmImportPane extends PaneBase {
             for (let colidx = 0; colidx < columns.length; colidx++) {
                 const column = this.getFieldID(columns[colidx]);
                 if(row.values[colidx]) {
+                    if(typeof(row.values[colidx]) !== 'string') {
+                        row.values[colidx] = str(row.values[colidx]);
```

Comment:
What is `str()`?

Reply:
Fixed this typo. It appears to work but I still haven't been able to reproduce the error.